backend/src/main/validation/PromotionValidation.ts

```diff
@@ -21,6 +21,9 @@ export class PromotionValidation {
     name: Joi.string().required(),
     description: Joi.string().required(),
     expirationDate: Joi.date().required(), // note incoming format is a string, and Joi will automatically convert to type Date
+    lat: Joi.number().required(),
+    lon: Joi.number().required(),
```

Comment:
I remember you mentioned that if the input was a string, it would convert it to a number automatically. If that's expected behaviour, maybe you can put a comment above explaining that?

Reply:
Yeah sure!!